lib/cli-engine.js

```diff
@@ -273,6 +273,17 @@ function CLIEngine(options) {
      */
     this.options = assign(Object.create(defaultOptions), options || {});
 
+    /**
+     * cache used to not operate on files that haven't changed since last successful
+     * execution (e.g. file passed with no errors and no warnings
+     * @type {Object}
+     */
+    this.fCache = require("file-entry-cache").create(path.resolve(".eslintcache"));
```

Comment:
And we need to consider if this should be private (begin with `_`). Do we want to allow consumers of `CLIEngine` to alter this?